wiki/BanchoBot/en.md

```diff
@@ -1,154 +1,168 @@
+13:05 BanchoBot: hacked, hard, hebrew, help, hitsound-library, hl, how-to-bn, how-to-get-modded, how-to-gmt, how-to-play, how-to-rank, hungarian
+13:05 BanchoBot: hyperdash, ignore, import, Indonesian, inheriting, irc, Italian, japanese, keys, kiai, korean, kudosu 
+13:05 BanchoBot: lag, language, language-specific, level, tines, link, lobby, localisation, long-link, long-text, toyed, mac 
+13:05 BanchoBot: malaysian, map-this, mapping, mat, mentoring, modhetp, modqueue, modreqs, mptimeroo, multi invite, multiaccounting, multiplayer 
+13:05 BanchoBot: mwc, nat, nightcore, noel, norwegian, np, nsfw, old-maps, osu, osu!droid, osugablet, osu!talk 
+13:05 BanchoBot: osucore, osumania, osustream, osz, OSZ2, owc, peppy, percolate, perfect, ping, pippi, piracy 
+13:05 BanchoBot: play-shortcuts, pm, polish, portuguese, pp, preview, problem, qat, qualified, ranked, ranking-criteria, redname 
+13:05 BanchoBot: report, report-abuse, request, respect, roll-abuse, romaji, romanian, room, rp, rules, russian, sauce 
+13:05 BanchoBot: sb, scorevz, scoring, seitseid, silence, skandinavian, skinning, sleep, songs-not-appearing, spaces, spam, spanish 
+13:05 BanchoBot: spectate, spoiler, status, store, submit, suck, supporter, supporter-please, swedish, tablet, tag, taiko 
+13:05 BanchoBot: taiko-mapping, taiko-maps, thai, topic, training, tsya, turkish, tutorial, twc, ukrainian, unicode, upload-cap 
+13:05 BanchoBot: utfa, videogames, vietnamese, welcome, when, wiki, zallius 
 ```
 
+An example of a moderator using the `!faq` command in a public chat channel is shown below:
+
 ```
-Tama: 你好
+pippi: 你好
 Yuzu: !faq chinese
 BanchoBot: 中文用户请点击 #chinese 以进入中文频道进行交流。
 BanchoBot: #osu 是英文专属频道，如果接获此讯息后继续在 #osu 内以中文交谈，管理员有权利禁言。
-Tama: ok
 ```
 
-### REPORT
-
-*Notice: If you want to report a moderator, send an email to [support@ppy.sh](mailto:support@ppy.sh) for moderator complaints.*
+### Report
 
-*Note: You can also use the user card to report a user.*
+*For information about what is worth reporting, see [Reporting Bad Behaviour](/wiki/Reporting_Bad_Behaviour).*
 
 ```
-!REPORT <user> <reason>
+!report <user> <reason>
 ```
 
-*For information about what is worth reporting, see [Reporting Bad Behaviour](/wiki/Reporting_Bad_Behaviour).*
-
 *Notice: When entering the username, replace spaces with underscores (`_`).*
 
-Sends a report to the [Global Moderation Team](/wiki/Global_Moderation_Team) or the [Language Moderators](/wiki/Language_Moderators).
-
-#### Usage Example
+`!report` sends a report to the [Global Moderation Team](/wiki/Global_Moderation_Team) (GMT) or the [Language Moderators](/wiki/Language_Moderators); it is also possible to report a user by right-clicking on said user's user card. Although neither of these methods are viable for reporting moderators. To report a moderator, send an email to [support@ppy.sh](mailto:support@ppy.sh). An example of reporting a normal user through BanchoBot is shown below:
```

Comment:
- Language Moderators team has been merged with GMT quite some time ago
- again, erroneous usage of "Although"